features/config/dao/impl/src/main/java/org/opennms/features/config/dao/impl/util/JSONObjectSerialIzer.java

```diff
@@ -0,0 +1,55 @@
+/*******************************************************************************
+ * This file is part of OpenNMS(R).
+ *
+ * Copyright (C) 2019-2019 The OpenNMS Group, Inc.
+ * OpenNMS(R) is Copyright (C) 1999-2019 The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is a registered trademark of The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ *
+ * OpenNMS(R) is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with OpenNMS(R).  If not, see:
+ *      http://www.gnu.org/licenses/
+ *
+ * For more information contact:
+ *     OpenNMS(R) Licensing <license@opennms.org>
+ *     http://www.opennms.org/
+ *     http://www.opennms.com/
+ *******************************************************************************/
+package org.opennms.features.config.dao.impl.util;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import com.fasterxml.jackson.databind.ser.std.StdSerializer;
+import org.json.JSONObject;
+
+import java.io.IOException;
+
+public class JSONObjectSerialIzer extends StdSerializer<JSONObject> {
+    public JSONObjectSerialIzer() {
+        this(null);
+    }
+
+    public JSONObjectSerialIzer(Class<JSONObject> t) {
+        super(t);
+    }
+
+    @Override
+    public void serialize(
+            JSONObject value, JsonGenerator jgen, SerializerProvider provider)
+            throws IOException {
+        jgen.writeStartObject();
+        String tmp = value.toString();
+        jgen.writeRaw(tmp.substring(1, tmp.length() - 1));
```

Comment:
just curious: why do we need that? Maybe put a comment?

Reply:
comment added